config.py

```diff
@@ -81,7 +81,6 @@ class Config:
     RM_CASE_SERVICE_PORT = os.getenv('RM_CASE_SERVICE_PORT', 8171)
     RM_CASE_SERVICE_PROTOCOL = os.getenv('RM_CASE_SERVICE_PROTOCOL', 'http')
     RM_CASE_SERVICE = f'{RM_CASE_SERVICE_PROTOCOL}://{RM_CASE_SERVICE_HOST}:{RM_CASE_SERVICE_PORT}/'
-    RM_CASE_POST = '{}cases/{}/events'
 
```

Comment:
Not sure this needs to be in global app config, it could just be a constant in case_service.py

Reply:
I don't get what you mean.  It was there before and I'm removing it as it's indirection for indirections sake.  It was also used 1 time in the whole of secure message (not including the tests where it's used twice).

Reply:
Sorry my bad, read it quickly and thought you'd added it.  You've done what I wanted anyway